### DISCUSSION TITLE
Subgrain boundaries

### DISCUSSION THREAD

**Discussion by Sabalei** [idaholab/moose#28899] [Q&A Modules: Phase field]
Dear Moose Team,

I am a new user of Moose and currently learning how to utilize it effectively.

I wonder if there is a way to identify and visualize subgrain boundaries from the EBSDReader.

I highly appreciate any help i can get.

**Comment by GiudGiud** [comment]
Hello

Should you see those from the values of the order parameters? Maybe I m mistaking grain and sub-grain boundaries.
My understanding is that you load the Mesh with the EBSDMG then use the EBSDReader to help set all the initial conditions, after which you can just visualize the system using the Exodus output

Guillaume
